File: barcosEnContienda/src/gestorAplicacion/Personas/CapitanFantasma.java
package gestorAplicacion.Personas;

import gestorAplicacion.Embarcaciones.Barco;

public class CapitanFantasma extends Capitanes {
	public CapitanFantasma(String apodo, int liderazgo, int elocuencia) {
		super(apodo, liderazgo, elocuencia);
	}
	
	public void yells() {
		System.out.print("Booooo");
	}
}

File: practica-2-grupo-1-equipo-7/src/gestorAplicacion/personas/capitanFantasma.py
from gestorAplicacion.embarcaciones.barco import Barco
from gestorAplicacion.personas.capitanes import Capitanes

class CapitanFantasma(Capitanes):
    
    def __init__(self,apodo,liderazgo,elocuencia):
        super().__init__(apodo,liderazgo,elocuencia)

    def yells(self):
        return "Fantasma \n"